recipes-core/images/intel-aero-image.bb

```diff
@@ -56,6 +56,7 @@ GRUB_TIMEOUT = "3"
 
 # librealsense
 IMAGE_INSTALL += "librealsense"
+IMAGE_INSTALL += "librealsense-graphical-examples"
```

Comment:
Let me update the PR to remove the /lib64 link (I'm not sure why it's there) and run the relasense examples.